test/NumberConverterTest.java

```diff
@@ -0,0 +1,63 @@
+public class NumberConverterTest {
+
+    public static void main(String[] args) {
+        NumberConverterTest.inttostrTest();
+        NumberConverterTest.strtointTest();
+        NumberConverterTest.dbltostrTest();
+        NumberConverterTest.strtodblTest();
+
+    }
+
+    public static void comparison(String result1, String result2,String iftrue, String ifrail) {
+        if (result2.equals(result1)){
+            System.out.println(iftrue);
+        }else {
+            System.out.println(ifrail);
+        }
+
+    }
+
+    public static void comparison(double result1, double result2,String iftrue, String ifrail) {
+
+        if (result1 == result2){
+            System.out.println(iftrue);
+        }else {
+            System.out.println(ifrail);
+        }
+
+    }
+
+    public static void inttostrTest() {
+
+        String result1 = NumberConverter.inttostr(15);
+        String result2 = NumberConverter.inttostr(-15);
+        comparison("15",result1,"Test1 positive: method inttostr test is successful","Test1 positive: method inttostr test is fail");
```

Comment:
don't use "Test1" and "Test2", just leave "method {methodName} test is successful"